rskj-core/src/main/java/co/rsk/peg/PegUtilsLegacy.java

```diff
@@ -222,12 +223,15 @@ protected static boolean isValidPegInTx(
                         logger.warn("[isValidPegInTx] {}", message);
                         throw new ScriptException(message);
                     }
-                    Script inputStandardRedeemScript = redeemScriptParser.extractStandardRedeemScript();
+
+                    List<ScriptChunk> inputStandardRedeemScriptChunks = redeemScriptParser.extractStandardRedeemScriptChunks();
+                    Script inputStandardRedeemScript = new ScriptBuilder().addChunks(inputStandardRedeemScriptChunks).build();
                     if (activeFederations.stream().anyMatch(federation -> getFederationStandardRedeemScript(federation).equals(inputStandardRedeemScript))) {
                         return false;
                     }
 
-                    Script outputScript = ScriptBuilder.createP2SHOutputScript(inputStandardRedeemScript);
+                    Script inputRedeeemScript = new ScriptBuilder().addChunks(inputStandardRedeemScriptChunks).build();
```

Comment:
typo here, inputRed**eee**mScript

Reply:
btw, this variable is the same as `inputStandardRedeemScript`, declared above.

Reply:
Done! Thanks, Juli!